cli/cmd/inventory.go

```diff
@@ -0,0 +1,38 @@
+package cmd
```

Comment:
Should this cmd be a service-plans command instead? i.e., is inventory just service-plans list?